### PR TITLE
schemars example for CollectionsResponse, ScrollResult, SnapshotDescription, AliasDescription

### PR DESCRIPTION
**Pull request by kartik-gupta-ij**
[Fern api demo docs link](https://qdrant-preview-a46b1a55-4c80-4748-ba62-2f844d5afa4f.docs.buildwithfern.com/master/api-reference)

### CODE REVIEW

**Comment by xzfc** [comment]
Please re-run `tools/generate_openapi_models.sh` to make test-consistency pass.

File: lib/api/src/grpc/models.rs [hunk 1]
```diff
@@ -51,8 +51,27 @@ pub struct CollectionDescription {
...
+            },
+            CollectionDescription {
+                name: "medium-title".to_string(),
+            },
+            CollectionDescription {
+                name: "medium-text".to_string(),
+            },
+        ],
+    }
+}
```
Comment (xzfc) [inline lib/api/src/grpc/models.rs:71]: Style nit: add newline.

File: lib/collection/src/operations/types.rs [hunk 2]
```diff
@@ -388,11 +388,38 @@ impl Default for ScrollRequestInternal {
...
+        },
+        api::rest::Record {
+            id: PointIdType::NumId(44),
+            payload: Some(Payload(payload_map_2)),
+            vector: Some(VectorStruct::Single(vec![0.75, 0.640625, 0.8945])),
+            shard_key: Some("region_1".into()),
+            order_value: Some(23.into()),
+        },
+    ]
+}
```
Comment (xzfc) [inline lib/collection/src/operations/types.rs:416]: Style nit: add newline.

**Review by xzfc — CHANGES_REQUESTED** [review]

File: lib/collection/src/operations/types.rs [hunk 3]
```diff
@@ -388,11 +388,39 @@ impl Default for ScrollRequestInternal {
...
+    payload_map_1.insert("city".to_string(), Value::String("London".to_string()));
+    payload_map_1.insert("color".to_string(), Value::String("green".to_string()));
+
+    let mut payload_map_2 = Map::new();
+    payload_map_2.insert("city".to_string(), Value::String("Paris".to_string()));
+    payload_map_2.insert("color".to_string(), Value::String("red".to_string()));
+
+    vec![
+        api::rest::Record {
+            id: PointIdType::NumId(43),
```
Comment (generall) [inline lib/collection/src/operations/types.rs:402]: Default example value for `PointIdType` is 42, so it results in a response example where `next_page_offset` is lower than Ids in the `points` (43 and 44). However, this is not possible, scroll API should always return IDs lower than `next_page_offset`. Propose to replace with `40` and `41`

File: lib/collection/src/operations/types.rs [hunk 4]
```diff
@@ -388,11 +388,39 @@ impl Default for ScrollRequestInternal {
...
+            vector: Some(VectorStruct::Single(vec![0.875, 0.140625, 0.897_6])),
+            shard_key: Some("region_1".into()),
+            order_value: Some(23.into()),
+        },
+        api::rest::Record {
+            id: PointIdType::NumId(44),
+            payload: Some(Payload(payload_map_2)),
+            vector: Some(VectorStruct::Single(vec![0.75, 0.640625, 0.8945])),
+            shard_key: Some("region_1".into()),
+            order_value: Some(23.into()),
```
Comment (generall) [inline lib/collection/src/operations/types.rs:413]: `order_value` is only present if we explicitly specify order-by. This is not a default behavior and query examples are not going that.

File: lib/collection/src/operations/snapshot_ops.rs [hunk 5]
```diff
@@ -81,7 +82,17 @@ pub struct SnapshotRecover {
     pub api_key: Option<String>,
 }
 
+fn snapshot_description_example() -> SnapshotDescription {
+    SnapshotDescription {
+        name: "prefix-cache".to_string(),
```
Comment (timvisee) [inline lib/collection/src/operations/snapshot_ops.rs:87]: Names are usually in a format like this:

```suggestion
        name: "my-collection-3766212330831337-2024-07-22-08-31-55.snapshot".to_string(),
```

File: openapi/openapi.lib.yml [hunk 6]
```diff
@@ -22,8 +22,10 @@ default:
             type: number
             format: float
             description: Time spent to process this request
+            example: 1.25
```
Comment (timvisee) [inline openapi/openapi.lib.yml:25]:
```suggestion
            example: 0.002
```

File: openapi/openapi.lib.yml [hunk 7]
```diff
@@ -51,8 +53,10 @@ default:
...
```
[the remark this answers is not in the document]
Comment (timvisee) [reply openapi/openapi.lib.yml:56]:
```suggestion
            example: 0.002
```

**Review by generall — APPROVED** [review]

**Pull request merged**